src/diffpy/srfit/equation/builder.py

```diff
@@ -662,13 +662,16 @@ def __wrapSrFitOperators():
     """
     opmod = literals.operators
     excluded_types = set((opmod.CustomOperator, opmod.UFuncOperator))
-    # check if opmod member should be wrapped as OperatorBuilder
-    is_exported_type = lambda cls: (
-        inspect.isclass(cls)
-        and issubclass(cls, opmod.Operator)
-        and not inspect.isabstract(cls)
-        and cls not in excluded_types
-    )
+
+    def is_exported_type(cls):
```

Comment:
`__wrapSrFitOperators` is a public function and I decided to make `is_exported_type` nested as well to encapsulate.